marcdown_project/frontend/src/pages/Home.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+
+class Home extends Component {
+  render() {
+    return (
+      <div>
+        <div>
+          <span>Tags</span>
+          <input type="text" />
+          <div>
+            {/* tag list */}
+          </div>
+        </div>
+        <div>
+          <h1>My notes</h1>
+          <div></div>
+          <h1>Shared with me</h1>
+          <div></div>
+        </div>
```

Comment:
Could add a "favorites" line

Reply:
That's temporary text anyway, will be changed when working on the page itself